.reuse/dep5

```diff
@@ -89,6 +90,10 @@ Files: vendor/github.com/go-openapi/swag/*
 Copyright: 2015 go-swagger
 License: Apache-2.0
 
+Files: vendor/github.com/go-task/slim-sprig/*
```

Comment:
is this addition related to this PR ?

Reply:
I guess this was a dependency that was never documented properly. 

When I run the tests in my fork the reuse tests were running (and here are not but I am not aware if it because I merging from my fork to the upstream) and the reuse test was failing because of the missing license for `slim-sprig`. I added this so that the test will be in a green state when the gh-action is merged.